scopes/component/component/component.ts

```diff
@@ -105,6 +106,22 @@ export class Component {
     }
   }
 
+  async getLogs(filter?: { type?: string; offset?: number; limit?: number }) {
+    const allLogs = await this.factory.getLogs(this.id);
```

Comment:
I'm wondering if it's not better to use the new methods to get this data (component.snapsIterable), to get the snaps, and create the logs out of them